src/instructlab/model/evaluate.py

```diff
@@ -334,6 +344,20 @@ def launch_server(
     default="",
     help="TLS client certificate password for model serving.",
 )
+@click.option(
+    "--backend",
+    type=str,
+    default=backends.VLLM,
+    show_default=True,
+    help="serving backend to use for evaluation. Options are vllm and llama-cpp.",
```

Comment:
```suggestion
    help="serving backend to use for evaluation. Options are vllm and llama-cpp.",
```
Can we just make this a `click.Choice`?

Reply:
no because the serve config does not pick a default (it probably should)

Reply:
The serve config does not pick a default because the backend is auto-detected and set later down the road. We can use `click.Choice`, this is how I did it here https://github.com/instructlab/instructlab/pull/1561 based on @tiran's suggestion.

Reply:
hm, when using click.choice, I get an error. My config.yaml has an empty string for the backend?

Reply:
`backend: null`

Reply:
mine doesn't seem to default to that, is there a recent patch changing this behavior?

```
serve:
  backend: ''
```

Reply:
didn't end up editing the cfg yaml, also used a click.choice